Unit-01/03-templating/app.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, render_template, request
```

Comment:
Can you open up a separate PR for 03? Also, it looks like there are some files you can remove (e.g. HTML files that are not in the templates folder).